.github/workflows/e2e-main.yml

```diff
@@ -140,19 +140,14 @@ jobs:
       env:
         TERM: xterm
 
-    - name: Run Cypress tests on ${{ matrix.folder }}
-      if: ${{ github.ref != 'refs/heads/master' }}
-      run: |
-        yarn run test-cypress-run \
-          --folder ${{ matrix.folder }}
-      env:
-        TERM: xterm
```

Comment:
What was the reason(ing) behind deleting this chunk of code? Did we decide not to run E2E tests on a release branch?